api/libproject/devstart.rb

```diff
@@ -1265,13 +1293,15 @@ def create_local_es_index(cmd_name, *args)
     '--delete-indices'
     # Gradle args need to be single-quote wrapped.
   ]).map { |f| "'#{f}'" }
-  common.run_inline %W{gradle elasticSearchIndexer -PappArgs=['create',#{create_flags.join(',')}]}
+  ServiceAccountContext.new(TEST_PROJECT).run do
```

Comment:
I'm confused about the hard-coded TEST_PROJECT. Shouldn't this refer to opts.env project name as well?

Reply:
Yes, thanks. Did that with a fallback to TEST_PROJECT for local runs.